### PR TITLE
chore: replace - with _

### PR DESCRIPTION
**Pull request by masci**
Part of #112

### CODE REVIEW

**Review by anakin87 — APPROVED** [review]
All good!

File: .github/labeler.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ integration:elasticsearch:
 
 integration:google-vertex:
```
Comment (silvanocerza) [inline .github/labeler.yml:17]: Should we change the labels too? 🤔

Comment (masci) [reply .github/labeler.yml:17]: we're going to have two naming conventions now:
- hyphens for package names: `google-vertex-haystack`
- underscores for folder names: `integrations/google_vertex/**`

I think it depends on what are you targeting with (for example in this case) a label: the folder in the source layout, or the "integration project"?

Comment (silvanocerza) [reply .github/labeler.yml:17]: Ah right, makes sense.

**Review by silvanocerza — APPROVED** [review]

**Pull request merged**